README.md

```diff
@@ -84,8 +84,8 @@ Cache pruning flow:
 
 1. Get all files from `baseDir`.
 2. Sort them by `lastModified`, oldest first.
-3. Get total size in Bytes by `files.reduce` array method.
-4. Check total size from step #3 and subtract `cacheLimit` value.
+3. Get total size in Bytes by using `reduce` array method.
+4. Check total size from step 3 and subtract `cacheLimit` value.
 5. Run a while loop and keep deleting files so long as the current cache size (step 4) is larger than the size required to trigger cache pruning (`cacheLimit` value).
```

Comment:
Plz add _All steps above will run only if the image is not already cached, it runs after downloading a new image into the cache._